src/validation.cpp

```diff
@@ -2724,6 +2736,11 @@ bool PreciousBlock(CValidationState& state, const CChainParams& params, CBlockIn
     return g_chainstate.PreciousBlock(state, params, pindex);
 }
 
+/**
```

Comment:
The usual convention for method documentation in this project is to add the comment in the class definition, not above the implementation. This makes it easier to read the API documentation in one place when looking at the class.
That said, as both are in the `.cpp` file for this class I don't think that's much of an issue here.

Reply:
Fixed, thanks.